plugins/report-portal/src/utils/isReportPortalAvailable.ts

```diff
@@ -0,0 +1,9 @@
+import { Entity } from '@backstage/catalog-model';
+
+export const isReportPortalAvailable = (entity: Entity): boolean => {
+  return Boolean(
+    entity.metadata.annotations?.['reportportal.io/project-name'] &&
+      entity.metadata.annotations?.['reportportal.io/launch-name'] &&
+      entity.metadata.annotations?.['reportportal.io/host'],
```

Comment:
The host annotation should be optional. And if it's not provided, the plugin should use the first available integration from the app-config.

Since in most cases, there will only be 1 report portal integration. And in case they have multiple, they can add the host annotations as needed.

Reply:
removed it from here, and added check in ReportPortalCard component